proxy/src/main/java/com/wavefront/agent/listeners/otlp/OtlpMetricsUtils.java

```diff
@@ -105,6 +123,47 @@ private static List<ReportPoint> fromOtlpRequest(
     return wfPoints;
   }
 
+  @VisibleForTesting
+  static List<KeyValue> replaceServiceNameKeyWithServiceKey(List<KeyValue> attributes) {
+    Optional<KeyValue> serviceNameAttr =
+        attributes.stream()
+            .filter(kv -> kv.getKey().equals(OtlpTraceUtils.OTEL_SERVICE_NAME_KEY))
+            .findFirst();
```

Comment:
Can thsi be replaced by `OtlpTraceUtils.getAttrByKey()`?

Reply:
Yes